src/main/webapp/components/RequestClarificationDialog.js

```diff
@@ -41,8 +52,8 @@ export const RequestClarificationDialog = hh(class RequestClarificationDialog ex
   validateClarification() {
     let isValid = false;
     if (this.props.linkClarification === true) {
-      if (!isEmpty(this.state.clarification) && !isEmpty(this.state.pm) && 
-          !isEmpty(this.state.pm[0]) && !isEmpty(this.state.pm[0].key)) {
+      if (!isEmpty(this.state.clarification) && !isEmpty(this.state.pm) &&
```

Comment:
Please verify if we can use lodash's _.isEmpy here. Lodash is being imported where this isEmpty is defined anyway (src/main/webapp/util/Utils.js)

Reply:
I think it's OK to use this `isEmpty` and not `_.isEmpty`